src/components/executive-education-2u/UserEnrollmentForm.jsx

```diff
@@ -70,6 +80,7 @@ const UserEnrollmentForm = ({
           dateOfBirth: values.dateOfBirth,
         },
         termsAcceptedAt: toISOStringWithoutMilliseconds(new Date(Date.now()).toISOString()),
+        ...(enableDataSharingConsent ? { dataShareConsent: true } : {}),
```

Comment:
[curious] Should we be relying on `values.dataSharingConsent` here rather than always hardcoding to `true`?